src/components/PostInfo/PostInfo.jsx

```diff
@@ -1 +1,21 @@
-export const PostInfo = () => <>Put the post here</>;
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+
+import './PostInfo.scss';
+
+export const PostInfo = ({ post: { title, body, user = false, comments } }) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">{title}</h3>
+
+      <p>
+        {' Posted by  '}
+        <UserInfo user={user} />
```

Comment:
<img width="560" alt="image" src="https://github.com/user-attachments/assets/e89aea0f-9495-4f04-85f2-aaecb8a3341c">
user can be null, let's add the check here
```suggestion
        {user && <UserInfo user={user} />}
```